src/styles/blocks/card.scss

```diff
@@ -0,0 +1,54 @@
+.card {
+  box-sizing: border-box;
+  position: relative;
+  text-align: justify;
+  width: 200px;
```

Comment:
The card width is correctly set to 200px, which includes the border. This aligns with the task requirements.